rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1554,6 +1554,21 @@ String loginOrRenewSession(@HeaderParam("sessionid") String sessionId, @FormPara
 	String loginOrRenewSession(@HeaderParam("sessionid") String sessionId, @MultipartForm LoginForm multipart)
 			throws KeyException, SchedulerRestException, LoginException, NotConnectedRestException;
 
+    /**
+     * Get the login string from the {@code sessionId} if it exists
+     * 
+     * @param sessionId given sessionId
+     * @return login string
```

Comment:
user login associated to sessionId. What happen if session id is invalid or has expired. Please add a brief explanation.